about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
+
+### Data processing & munging
+
+Data munging tools are tools that support processing data discussed above. This
+category focuses on tools for handling data in specific  formats that scientists
+may be interested in working with. These data may also be generated from
+scientific workflows or exported from instruments and wearables.
+
+* Examples: [physcraper](https://github.com/pyOpenSci/software-submission/issues/26)
+
+
+### Data deposition
+
+Tools for depositing data into scientific research repositories.
+
+* Examples: [This is an example from rOpenSci - eml](https://github.com/ropensci/software-review/issues/80)
+
+### Data validation and testing:
+
+Tools that enable automated validation and checking of data quality and
+completeness. These tools should be able to support scientific workflows.
+
+* Example: [pandera](https://github.com/pyOpenSci/software-submission/issues/12)
+
+### Scientific software wrappers
+
+Scientific software wrappers refer to packages that provide a Python interface
+for existing scientific packages written in other languages.
+
+These programs must be specific to scientific domains and not general computing
+utilities. Wrappers must provide significant added value to the scientific
+ecosystem be it in data handling, or improved installation processes for
+Python users.
+
+We strongly encourage submissions that wrap tools that are open-source and
+using an OSI-approved license. Exceptions will be evaluated case-by-case,
+considering whether open-source options exist.
+
+<!-- TODO: need an example for this category -->
+* Examples: We don't have a package in this category yet - *Could be your package?*
+
+### Workflow Automation & versioning
+Tools that automate and link together workflows and as such support
+reproducible workflows. These
+tools may include build systems and tools to manage continuous integration.
+This also includes tools that support version control.
+
+<!-- TODO: marting git package is a good example but it's needs to be sunsetted as it's no longer maintained. hamilton is just starting review but would fit
+well here  - the link below for it is a presubmission not a review   -->
+* Examples: [Hamilton - currently under review](https://github.com/pyOpenSci/software-submission/issues/74), martin's git package (that is no longer maintained.)
+
+### Citation management and bibliometrics:
+
+Tools that facilitate managing references, such as for writing manuscripts,
+creating CVs or otherwise attributing scientific contributions, or accessing,
+manipulating or otherwise working with bibliometric data. (Example: [Example from rOpenSci - RefManageR](https://github.com/ropensci/software-review/issues/119))
+
+### Data visualization & analysis
+These are packages that enhance a scientists experience visualizing and
+analyzing data.
+
+* Examples: [PyGmt - (also spatial and data munging)](https://github.com/pyOpenSci/software-submission/issues/43),
+
+### Database software bindings
```

Comment:
Really? I find this a little surprising 🤷

Reply:
oh - tell me how your interpreted this please. 

I think i can write more... but i'm thinking about tools that connect to databases and allow for data transfer and manipulation. I suspect there are many storing data in cloud based or other databases, postgres, mysql, even airtable etc. 

Let me know your thoughts so i can better clarify this! maybe software bindings are not the right words

Reply:
Of course science needs data, and databases are great tools to deal with data. I'm curious about example Python packages that could be in scope. Several tools are more "business-y", "web stack-y", or "enterprise-y", but it's a wide gamut that can also include science-y tools. Would any of these be in scope (found randomly via google search)?
- https://python.libhunt.com/categories/241-database-drivers
- https://gitee.com/shine_forever/awesome-python#database

If the developers of a database tool feel that they belong in pyOpenSci and not some other community, I don't want to discourage them. My impression though is that many don't come from scientific backgrounds, but this is not universally true.

Would [Ibis](https://github.com/ibis-project/ibis) be an example package that would be within scope?

Reply:
Ok i didn't respond to this because i wasn't sure i had an answer (and i still dont!! 😆 ) i am going to see how ropensci handles "general utilities" because i'm struggling with defining what is general - you are right there is a line that is easy to lose track of between what is a "science-y" tool vs a more general tool!! i think this relates to your question about about wrappers...

Reply:
i think ibis would be in scope because it is thinking about dataframes and references dask, dplyr (r) but those other packages in the links you have above i think would be out of scope / too general for us to review. for instance we wouldn't ever review the requests library i think? that is super general. Similarly a mysql library seems too general too unless it interfaces with a specific data format and pushes to a database... i'm not exactly sure how to use words to describe the difference between the two however? 

maybe we are back to clear scientific focused application? or that targets scientific users? thoughts?